experiments/AMIP/components/atmosphere/climaatmos_init.jl

```diff
@@ -97,8 +100,11 @@ function get_field(sim::ClimaAtmosSimulation, ::Val{:liquid_precipitation})
     sim.integrator.p.precipitation.col_integrated_rain .* ρ_liq # kg/m^2/s
 end
 function get_field(sim::ClimaAtmosSimulation, ::Val{:snow_precipitation})
-    ρ_liq = CAP.ρ_cloud_liq(sim.integrator.p.params)
-    sim.integrator.p.precipitation.col_integrated_snow .* ρ_liq  # kg/m^2/s
+    # TODO: use this upon Atmos 0.19.3 release
```

Comment:
it will be v0.20 not 0.19.3 (but same idea) :)

Reply:
Nice! :)

Reply:
reminder: https://github.com/CliMA/ClimaCoupler.jl/issues/596